wiki0/src/server/controllers/authtoken.ts

```diff
@@ -1,10 +1,24 @@
 import { Router } from 'express';
 import prisma from '../../../prisma/client';
+import createRedisClient from '../redis/client';
 
 const controller = Router();
 
+const getTokenFromRedis = async () => {
+  try {
+    const redisClient = await createRedisClient();
+    await redisClient.connect();
+    const token = await redisClient.get('jag_subject_token');
```

Comment:
actually I'm confused, are the wiki server and auth server sharing state?  I think the auth server should expose an http endpoint that the wiki server calls